__tests__/get-todo-config-test.ts

```diff
@@ -0,0 +1,167 @@
+import { getTodoConfig } from '../src';
+import { FakeProject } from './__utils__/fake-project';
+
+describe('get-todo-config', () => {
+  let project: FakeProject;
+
+  beforeEach(() => {
+    project = new FakeProject();
+    project.writeSync();
+  });
+
+  afterEach(() => {
+    project.dispose();
+  });
```

Comment:
This needs to account for tearing down the various "setupEnvVar"s (or tests bleed into each other)

Reply:
Each test is tearing down its own env vars, so there's no bleeding, unless you see one that's not correctly tearing things down.

Reply:
I added an explicit teardown util so it's more clear that's what's going on.